module/People/src/People/Controller/TaskStatsController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace People\Controller;
+
+use Application\Controller\OrganizationAwareController;
+use Application\Service\UserService;
+use People\Service\OrganizationService;
+use People\View\TaskStatsJsonModel;
+use TaskManagement\Controller\TasksController;
+use TaskManagement\Service\TaskService;
+use Zend\Validator\Date as DateValidator;
+
+class TaskStatsController extends OrganizationAwareController{
+	
+	protected static $collectionOptions = [];
+	protected static $resourceOptions = ['GET'];
+	
+	/**
+	 * @var TaskService
+	 */
+	private $taskService;
+	/**
+	 * @var UserService
+	 */
+	private $userService;
+	
+	public function __construct(OrganizationService $orgService, TaskService $taskService, UserService $userService){
+		parent::__construct($orgService);
+		$this->taskService = $taskService;
+		$this->userService = $userService;
+	}
+	
+	public function get($memberId){
+
+		if(is_null($this->identity())) {
+			$this->response->setStatusCode(401);
+			return $this->response;
+		}
+		$user = $this->userService->findUser($memberId);
+		if(is_null($user)){
+			$this->response->setStatusCode(404);
+			return $this->response;
+		}
+		
+		if(!$this->isAllowed($this->identity(), $user, 'People.User.taskMetrics')) {
+			$this->response->setStatusCode(403);
+			return $this->response;
+		}
+		$dateValidator = new DateValidator();
+
+		$endOn = new \DateTime();
+		$startOn = TasksController::getDefaultStartOn($endOn);
```

Comment:
Perché usi un metodo statico del `TasksController`? Non è importante tenere allineata l'implementazione dei due controller (le logiche potrebbero anche differire in futuro). Ti suggerirei di creare un metodo `private getDefaultStartOn` in questo controller